test/topology_custom/test_tablets.py

```diff
@@ -95,3 +96,45 @@ async def test_reshape_with_tablets(manager: ManagerClient):
     await log.wait_for("Reshape test.test .* Reshaped 32 sstables to .*", mark, 30)
     sstable_info = await manager.api.get_sstable_info(server.ip_addr, "test", "test")
     assert len(sstable_info[0]['sstables']) == number_of_tablets
+
+
+@pytest.mark.parametrize("direction", ["up", "down"])
+@pytest.mark.xfail(reason="Scaling not implemented yet")
+@pytest.mark.asyncio
+async def test_tablet_rf_change(manager: ManagerClient, direction):
+    cfg = {'enable_user_defined_functions': False,
+           'experimental_features': ['tablets', 'consistent-topology-changes']}
+    servers = await manager.servers_add(3, config=cfg)
+
+    cql = manager.get_cql()
+    res = await cql.run_async("SELECT data_center FROM system.local")
+    this_dc = res[0].data_center
+
+    if direction == 'up':
+        rf_from = 2
+        rf_to = 3
+    if direction == 'down':
+        rf_from = 3
+        rf_to = 2
+
+    await cql.run_async(f"CREATE KEYSPACE test WITH replication = {{'class': 'NetworkTopologyStrategy', '{this_dc}': {rf_from}}}")
+    await cql.run_async("CREATE TABLE test.test (pk int PRIMARY KEY, c int);")
+
+    logger.info("Populating table")
+    for _ in range(32):
+        await asyncio.gather(*[cql.run_async(f"INSERT INTO test.test (pk, c) VALUES ({k}, {k});") for k in range(64)])
```

Comment:
Why do it 32 times? The batch is the same in each iteration.

Reply:
Yikes. It was 32 tables in the test initially, then I left only one and forgot to clean this place. Will fix